README.md

```diff
@@ -75,6 +75,6 @@ Thank you to the following supporters:
 If you support us on Patreon and would like your name added, please get in touch or open a Pull Request.
 
 ## Note
-The purpose of the emulator is not to enable illegal activity.
+The purpose of the emulator is not to enable illegal activity. You can dump games by using [MaiDump](https://github.com/LioMajor/MaiDumpToolEN/releases)/[FAGDec](https://github.com/CelesteBlue-dev/PSVita-RE-tools/tree/master/FAGDec/build). You can find homebrew programs at [VitaDB](https://vitadb.rinnegatamante.it/).
```

Comment:
Maidump is now deprecated, please change to nonpdrm

Reply:
Fixed, the vita3k says to use MaiDump btw (as of two days ago)